src/components/UI/navbar/index.js

```diff
@@ -187,26 +176,13 @@ const Navbar = () => {
                   </Link>
                 )}
                 {isLoggedIn && (
-                  <div className={styles.userGreet}>
-                    <Link href={USER_PROFILE_URL}>
-                      <a>
-                        <div className={styles.userGreetMsg}>
-                          {isLoggedIn
-                            ? `Hello, ${userData.firstName}!`
-                            : `Hello, User!`}
-                        </div>
-                        <img
-                          className={styles.userProfilePic}
-                          src={
-                            isLoggedIn
-                              ? `${userData.profilePicture}`
-                              : `${DEFAULT_AVATAR}`
-                          }
-                          alt="Profile Pic"
-                        />
-                      </a>
-                    </Link>
-                  </div>
+                  <Dropdown
+                    isLoggedIn={isLoggedIn}
+                    USER_PROFILE_URL={USER_PROFILE_URL}
+                    userData={userData}
+                    DEFAULT_AVATAR={DEFAULT_AVATAR}
+                    SIGN_OUT_URL={SIGN_OUT_URL}
+                  />
```

Comment:
why do we have two dropdown components here, I saw it above as well

Reply:
Earlier there were two div elements are shown in the navbar file on is for web view and another for mobile view so I made the sperate component and called it twice.

Reply:
why we are using two navbars for different views?